### ISSUE TITLE
cant play

### ISSUE THREAD

**Issue by spammeurfou** [haxball/haxball-issues#35]
hello i play old haxball in safe mode with windows 7 because i have very bad pc, and it's impossible for me play with new haxball, i have bug all the time and it's inplayable, i hope u can change that, i'm not alone player playing in safe mode, so we need help plz

**Comment by Kozlov98** [comment]
Yes, that it's true. i posted a issue with that, i think that a optimizer or quality reduction, Basro can you do something?

**Comment by basro** [comment]
@spammeurfou, Using chrome go to the address chrome://gpu/ and copy what is in there.

**Comment by spammeurfou** [comment]
Note: To properly save this page, select the "Webpage, Complete" option in the Save File dialog.
Graphics Feature Status
Driver Bug Workarounds
Problems Detected
Version Information
Driver Information
Compositor Information
GpuMemoryBuffers Status
Diagnostics
... loading ...
None
Log Messages

**Comment by spammeurfou** [comment]
Data exported | 01/11/2017 à 12:52:16
-- | --
Chrome version | Chrome/62.0.3202.75
Operating system | Windows NT 6.1.7601 SP1
Software rendering list version | 13.13
Driver bug list version | 10.30
ANGLE commit id | e8ef2bc4bd01
2D graphics backend | Skia/62 e74b41c6c84638d5a9ee6d254a715bcd9e17c603-
Command Line | "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --flag-switches-begin --flag-switches-end

**Comment by spammeurfou** [comment]
Note: To properly save this page, select the "Webpage, Complete" option in the Save File dialog.
Graphics Feature Status
Canvas: Software only, hardware acceleration unavailable
CheckerImaging: Disabled
Flash: Software only, hardware acceleration unavailable
Flash Stage3D: Software only, hardware acceleration unavailable
Flash Stage3D Baseline profile: Software only, hardware acceleration unavailable
Compositing: Software only, hardware acceleration unavailable
Multiple Raster Threads: Disabled
Native GpuMemoryBuffers: Software only. Hardware acceleration disabled
Rasterization: Software only, hardware acceleration unavailable
Video Decode: Software only, hardware acceleration unavailable
Video Encode: Software only, hardware acceleration unavailable
WebGL: Hardware accelerated but at reduced performance
WebGL2: Unavailable
Problems Detected
Drivers older than 2009-01 on Windows are possibly unreliable: 72979, 89802, 315205
Disabled Features: flash_stage3d, gpu_compositing, panel_fitting, flash3d, gpu_rasterization, accelerated_2d_canvas, accelerated_video_decode, webgl2, accelerated_webgl, flash_stage3d_baseline, accelerated_video_encode
GPU rasterization should only be enabled on NVIDIA and Intel DX11+, and AMD RX-R2 GPUs for now.: 643850
Disabled Features: gpu_rasterization
Raster is using a single thread.
Disabled Features: multiple_raster_threads
Native GpuMemoryBuffers have been disabled, either via about:flags or command line.
Disabled Features: native_gpu_memory_buffers
Checker-imaging has been disabled via finch trial or the command line.
Disabled Features: checker_imaging


Data exported | 01/11/2017 à 12:52:16
-- | --
Chrome version | Chrome/62.0.3202.75
Operating system | Windows NT 6.1.7601 SP1
Software rendering list version | 13.13
Driver bug list version | 10.30
ANGLE commit id | e8ef2bc4bd01
2D graphics backend | Skia/62 e74b41c6c84638d5a9ee6d254a715bcd9e17c603-
Command Line | "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --flag-switches-begin --flag-switches-end


Initialization time | 0
-- | --
In-process GPU | true
Passthrough Command Decoder | false
Supports overlays | false
Sandboxed | false
GPU0 | VENDOR = 0x0000, DEVICE= 0x0000
Optimus | false
Optimus | false
AMD switchable | false
Desktop compositing | none
Driver vendor | Google Inc.
Driver version | 3.3.0.2
Driver date | 2017/04/07
Pixel shader version | 3.0
Vertex shader version | 3.0
Max. MSAA samples | 4
Machine model name |  
Machine model version |  
GL_VENDOR | Google Inc.
GL_RENDERER | Google SwiftShader
GL_VERSION | OpenGL ES 2.0 SwiftShader
GL_EXTENSIONS |  
Disabled Extensions |  
Window system binding vendor |  
Window system binding version |  
Window system binding extensions |  
Direct rendering | Yes
Reset notification strategy | 0x0000
GPU process crash count | 0


Tile Update Mode | One-copy
-- | --
Partial Raster | Enabled


ATC | Software only
-- | --
ATCIA | Software only
DXT1 | Software only
DXT5 | Software only
ETC1 | Software only
R_8 | Software only
R_16 | Software only
RG_88 | Software only
BGR_565 | Software only
RGBA_4444 | Software only
RGBX_8888 | Software only
RGBA_8888 | Software only
BGRX_8888 | Software only
BGRA_8888 | Software only
RGBA_F16 | Software only
YVU_420 | Software only
YUV_420_BIPLANAR | Software only
UYVY_422 | Software only

Diagnostics
... loading ...

**Comment by basro** [comment]
@spammeurfou, your computer has no hardware acceleration at all..
Please check if playing with a smaller window increases your fps, if it does then I could maybe make a low resolution setting for people with software only rendering.

**Comment by spammeurfou** [comment]
i have good fps, i have 60fps but i have all the time lags, i give u screen for u can understand my resolution

in haxball with adobe flash player : https://gyazo.com/273c14162b1e14caba8c7422dce44924

in haxball html 5 : https://gyazo.com/c7d7a12c72e77f24bd123e8a5e5fac92

with haxball adobe flash player, i have before good fluidity i can play good but since 1/2 months idk it's bad, i cant play good i have bad control... idk if u change this haxball or if my computer cant be good

and with haxball html 5 impossible for me to play, but it's strange because when i move my mouse i dont have lags, and when i stop move my mouse i have lags, idk if u can understand i have bad english

**Comment by spammeurfou** [comment]
and just safe mode is that ( https://www.lifewire.com/how-to-start-windows-7-in-safe-mode-2624540 ) i play with him because i cant play in windows normal, safe mode give me good fluidity before, now no =(
